contracts/smart-contract-wallet/paymasters/verifying/singleton/VerifyingSingletonPaymaster.sol

```diff
@@ -142,8 +146,6 @@ contract VerifyingSingletonPaymaster is BasePaymaster {
         paymasterNonces[userOp.getSender()]++;
```

Comment:
probably change to` ++paymasterNonces[userOp.getSender()]` according to gas optimizations reports

Reply:
done